RELEASE.md

```diff
@@ -13,6 +13,7 @@
 ## Major features and improvements
 
 ## Bug fixes and other changes
+* Added minimal, black-compatible flake8 configuration to the project template.
```

Comment:
```suggestion
```
@DmitriiDeriabinQB I tacked this on here, as #524 was merged. I suppose this should be added under 0.17.x breaking changes.